server/service/src/json_translate.rs

```diff
@@ -0,0 +1,86 @@
+use serde_json::Value;
+
+use crate::localisations::{GetTranslation, Localisations, TranslationError};
+
+const UNIQUE_TRANSLATE_KEY: &str = "T#";
+
+pub fn crawl_and_translate(
+    json: &mut Value,
+    translation_service: &Box<Localisations>,
+    user_language: &str,
+) -> Result<(), TranslationError> {
+    match json {
+        Value::String(text) => {
+            if let Some(key) = text.strip_prefix(UNIQUE_TRANSLATE_KEY) {
+                *text = translation_service.get_translation(
+                    GetTranslation {
+                        namespace: None,
+                        fallback: Some(key.to_string()),
+                        key: key.to_string(),
+                    },
+                    user_language,
+                )?
```

Comment:
Translation won't fail here on missing or incorrect translation keys, as we are providing a fallback.